clevertap-core/src/main/java/com/clevertap/android/sdk/AnalyticsManager.java

```diff
@@ -490,16 +496,21 @@ public void pushNotificationClickedEvent(final Bundle extras) {
                 @Override
                 public Void call() {
                     try {
-                        Logger.v("Received in-app via push payload: " + extras
-                                .getString(Constants.INAPP_PREVIEW_PUSH_PAYLOAD_KEY));
-                        JSONObject r = new JSONObject();
+                        String inappPreviewPayloadType = extras.getString(Constants.INAPP_PREVIEW_PUSH_PAYLOAD_TYPE_KEY);
+                        String inappPreviewString = (String) extras.get(Constants.INAPP_PREVIEW_PUSH_PAYLOAD_KEY);
```

Comment:
why are we not using getString() method?

Reply:
@shivamsharma2710

Reply:
It was not intended and a mistake. Updated the code. Please resolve conversation.